docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services:
+  simulation:
+    build:
+      context: .
+      dockerfile: simulationDockerfile
```

Comment:
Nein das stimmt schon. Da beide etwas anderes starten, haben beide ein anderes Dockerfile. Wobei man vermutlich auch ein einziges Dockerfile verwenden könnte...
Ich versuch das in den nächsten Tagen mal.